src/functionalTest/java/uk/gov/hmcts/reform/judicialapi/AuthorizationFunctionalTest.java

```diff
@@ -72,14 +70,16 @@ public void setUp() {
         log.info("Configured S2S secret: " + s2sSecret.substring(0, 2) + "************" + s2sSecret.substring(14));
         log.info("Configured S2S microservice: " + s2sName);
         log.info("Configured S2S URL: " + s2sUrl);
+        RestAssured.baseURI = judicialApiUrl;
+        RestAssured.useRelaxedHTTPSValidation();
+
 
-        IdamOpenIdClient idamOpenIdClient = new IdamOpenIdClient(configProperties);
-        log.info("idamOpenIdClient: " + idamOpenIdClient);
         /*SerenityRest.proxy("proxyout.reform.hmcts.net", 8080);
         RestAssured.proxy("proxyout.reform.hmcts.net", 8080);*/
 
+        IdamOpenIdClient idamOpenIdClient = new IdamOpenIdClient(configProperties);
+        log.info("idamOpenIdClient: " + idamOpenIdClient);

```

Comment:
logger do we needed?

Reply:
I may keep this for now as IDAM integration is the next step in development work for this service, but I'm happy to remove logging thereafter